src/app/certificates/certificates.component.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../assets/styles/variables";
+
+.certificates-content {
+  height: calc(100vh - #{$header-height});
+
+  .title-content {
+    margin-top: 3rem;
+    color: $cutty-sark-color;
+
+    .line {
+      width: 3.2rem;
+      height: $line-height;
+      background-color: $cutty-sark-color;
+    }
+    .title {
+      font-size: $font-size-xxxl;
+      letter-spacing: 0.2rem;
+      font-weight: 500;
+      b {
+        font-weight: 800;
+      }
+    }
+    p {
+      text-align: justify;
+    }
+  }
+
+  .carousel {
+    margin-top: 3rem;
+  }
+
+  img {
+    display: block;
+    width: 100%;
+  }
+
+  ::ng-deep span {
+    background-color: $mantle-color;
+    height: 15px;
+  }
+  ::ng-deep li {
```

Comment:
@Yuriy-27 you shouldn't write ::ng-deep multiple times,
a bit better to read if you write

```::ng-deep {
           span {...}
           li {...}
     }
```